Tensile/Common.py

```diff
@@ -1519,6 +1519,14 @@ def supportedCompiler(compiler: str) -> bool:
     "MinKForGSU":                   [16,32,64,128,256]
     }
 
+def getNameAbbreviation(name):
+  specialValues = {
+    'MACInstruction': '' # Conflicts with MatrixInstruction, but _MAD and _FMA should be enough differentiation for the kernel name.
+  }
+  if name in specialValues: return specialValues[name]
+  return ''.join([c for c in name if not c.islower()])
```

Comment:
Minor, but a generator expression here would eliminate the need to use an intermediary list
```python
return ''.join(c for c in name if not c.islower())
```

Reply:
I will update to develop and make this change